pykoi/retrieval/vectordb/epsilla.py

```diff
@@ -0,0 +1,169 @@
+"""Vector store Epsilla module"""
+import os
+import numpy as np
+import types
+
+from typing import List
+from langchain.embeddings.base import Embeddings
+from langchain.schema import BaseRetriever, Document
+from langchain.embeddings import OpenAIEmbeddings
+from sklearn.decomposition import PCA
+from pyepsilla import vectordb
+
+from pykoi.retrieval.vectordb.abs_vectordb import AbsVectorDb
+
```

Comment:
please setup flake and pep8 python linter (2 lines here). Otherwise, pr will fail due to linting error. You can use https://black.readthedocs.io/en/stable/ to help format each .py file.

Reply:
done, please check.